### ISSUE TITLE
Josh record squirrel narration and music and slogan

### ISSUE THREAD

**Issue by CIEspost** [zacck/cieproductions#28]
Note that we need to explain and make sure that all sound comes out binaural (spatial)
Note that ffmpeg method is not secure and if Josh can make easier make sure it can be added to video by means of injecting spatial metadata for youtube view 

- [ ] @CIEspost check what formats is needed for spatial audio metadata injector

**Comment by CIEspost** [comment]
@ApheleleM I am adding you to all the issues so that you can see what the process is and so that you can keep updated with the project, let me know if this is good with you?

**Comment by ApheleleM** [comment]
Thank you for adding Carla, You guys have a safe trip <3

**Comment by CIEspost** [comment]
@zacck please lets make a date with josh, and then please add the date here

**Comment by ApheleleM** [comment]
did you Guys Manage to meet with Josh?

**Comment by zacck-zz** [comment]
@ApheleleM  we are in Gauteng now we are making the plan to meet with him we will update you soon!

**Comment by CIEspost** [comment]
@ApheleleM we are meeting Josh tomorrow night and doing the sound recording at his place yeah!
@zacck 4pm in Joburg

**Comment by CIEspost** [comment]
show josh clip once uploaded https://youtu.be/nUruwpVkW2c (he can look at it so long and thing of what to say) @zacck it will be done uplaoded in a few hours, please send him the link then and ask him to watch from minute 3:00 and tell him this is what we want narration for (anything else you think we shoudl tell him about the script etc? @ApheleleM what do you think?)

**Comment by CIEspost** [comment]
send josh edited episode 1 he will add naaration and send through and we will discuss from there

**Comment by CIEspost** [comment]
@zacck i have uploaded the video and would like to send it to Josh so that he can start on the sound. Do you think we should call him? Where would be best to send him the link?

**Comment by CIEspost** [comment]
Note that we need to explain and make sure that all sound comes out binaural (spatial)
Note that ffmpeg method is not secure and if Josh can make easier make sure it can be added to video by means of injecting spatial metadata for youtube view

- [ ]  @CIEspost check what formats is needed for spatial audio metadata injector

- [ ] Adjust the script https://docs.google.com/spreadsheets/d/1saNmVXKBFeoQkK8oTsSVurfElvVBWmcaw7ed12XODSc/edit#gid=0

*NB - Remember to speak to Josh about the music as well

**Comment by zacck-zz** [comment]
@CIEspost  Yes please call him and get the ball rolling

**Comment by CIEspost** [comment]
here is the link we can send josh so long https://youtu.be/kxYRA_jg14U
@zacck please send him this and ask him when we can chat? he can look at the video so long, this is the one we need narration for, i added some comments on the video description on youtube

**Comment by CIEspost** [comment]
I am uplaoding the video to dropbox

**Comment by CIEspost** [comment]
@zacck I sent the 80mb video via wetransfer to jstarzbeat@gmail.com, dropbox didnt want to work 
josh emails:
jstarzacheus@hotmail.com
jstarzbeat@gmail.com

**Comment by CIEspost** [comment]
@zacck josh sent us a rough draft, we need to check it out, I have, I have quite some changes i would prefer, but I need your help here please let me know when you can do this, we need to send comments trhough to Josh asap

**Comment by zacck-zz** [comment]
@CIEspost  can we work on this after looking for Maria's Sim card ?

**Comment by CIEspost** [comment]
@zacck  its more important than the sim card_ its urgent

**Comment by zacck-zz** [comment]
@CIEspost let's call Josh as we head the laundry we can multitask as we get that done

**Comment by CIEspost** [comment]
@zacck ok, i just want to speak to you about his current voice message and the rough narration he did before we call him plz

**Comment by zacck-zz** [comment]
👍

**Comment by CIEspost** [comment]
Josh: " I'm gonna call him Ty-Ty,  "Ja nee Ek is ou Ty-Ty I always fly high high looking for something easy just to please me" kinda singing it... His interaction with the other Squirrel will be more cat calling and flirtatious, he whistles "Hey baby wanna taste my banana PAUSE Peel" (Obvious Play on words) Piel🙈he laughs saying come here jou lekka ding... And TyTy is definitely rivals with the doves," telling him to dala what you must or get off my turf "I don't want no pigeon, Homer chise back to V&A, die park is by my... Ja neh if you hang with Ty don't be kak be lekker soos n cracker and easy peazy like a Sunday morning koeksister 
🙈
I'll make more notes tomorrow when I start working i just wanted you guys to know this shits gonna be dope... I'll work the accent and settle into the character once I get into it..."

**Comment by CIEspost** [comment]
The brief @carla sent Josh:                         
[15:54, 8/2/2017] +27 83 480 9771: Thanks again Josh for all the work awesome spontaneous input!
Like I mentioned we wanted to see what is your first reaction to the squirrel’s personality, and we really enjoy the chewing sounds,  "lekker soos n cracker" and the banana theme to name but a few! So as I explained the main thing we feel needs more work on is making this video fit into the bigger picture of the webseries of 12 episodes. The webseries will be almost like a sitcom. There is no overarching storyline to the series, however each episode will have its own little story and the only thing all the episodes will have in common is the fact that it is urban animals. The first season of the series will feature urban animals that can be found in the city of cape town, the second season urban animals from Joburg and the 3rd urban animals from durban and so on.
So this specific video of the squirrel is going to be the first debut episode for the series. 
So the main thing we realised is that we need to create a little storyline for this episode. You see how brickleberry episode 1 is an intro to the park, episode 2 is about Rex erection, episode 3 about the hillbilly and so on… a sitcom works in that way that each episode has a little theme to it… its own story.                        
[16:06, 8/2/2017] +27 83 480 9771: So this webseries of ours (we need to find a name for it still) is about urban animals that talk. One of the main things we want for this series is that it finds a liking from human beings who live in cities. With the first season we want it to specifically relate to people from cape town. We want the series to relate to the struggles urban people. So let's think about an urban personality.
We are in south africa, so majority of people aren’t rich.
Cape Town is the hipster capital.
Cape Town is a city like any other, meaning that all people want and like and think of is fast cars, sex, booze and parties. 
This is how we came to the realization that the animals that feature in the series must be city slick bargain hunter hipsters.                        
[16:08, 8/2/2017] +27 83 480 9771: So for the cape town season,  we will introduce the viewers to certain animal characters. All of these characters will have cape townian personality. But above and beyond this cape townian  personality each type of animal will embody a specific type of character that can be found in cape town. So the easiest way we thought of doing this is to make these characters have names. So say for example the squirrel is called Taariq. Then if we make another video of a squirrel we can use the same type of personality again.                        
[16:10, 8/2/2017] +27 83 480 9771: So the reason why we are calling him Taariq is because we have this friend from Cape Town called 
Taariq who really makes us think of this squirrel in the video.                        
[16:16, 8/2/2017] +27 83 480 9771: Taariq is from Mitchells Plain. He is 24 years old, and a very accomplished computer geek who doesnt ever want to get out of his house. He is super chill - if he finds a bong he'll smoke the whole of it. Taariq speaks English most of the time, but when he tells a story he throws in some iconic Mitchells Plain one liners and catch-phrases; he loves telling stories by the way! Taariq is quite high most of the time, which means that he always has munchies, but in cape town all the places to get food in the city is super expensive, so Taariq is always looking for the best deal possible - this and the fact that he needs to make money every month to pay rent is the only reason why he would ever leave his humble hipster home.                        
[16:22, 8/2/2017] +27 83 480 9771: Some hints on the narration: 
Maybe Taariq comments on the Mosque in the background, the cars klapping past or the Hadida we hear yelling later on, I think it might be nice if we make him interact with his environment in a way that will make people feel like they are in cape town and not just any other city.
Please dont mention the words hapihour, we want the series to be a standalone product aside from the app
Because Taariq is high all the time, we feel that we maybe we can him talk while chewing? We really want him to talk alot, aka think verbal diarrhoea. So maybe it can sound like he is talking with food in his mouth maybe?                        How To Make A Talking Dog Video That Will Be Seen And Shared By MillionsOh, the things animals say in those funny clips on YouTube’s Talking Animals channel. The creator, Andrew Grantham, talks about how he crafts the videos that have made stars out of a bacon-loving dog, a lot of clever cats, and a beaver, too.
www.fastcompany.com
[16:30, 8/2/2017] +27 83 480 9771: Then lastly you are welcome to add to Taariq’s personality or change his name? We just used this to explain the basic idea we are going for. We think there is always more to add to the personality… let’s think of key things that will add more spice to the character. Perhaps we can write down what he likes and dislikes… like the colour blue? Or maybe he hates doves for some or other reason? Maybe in the realm of squirrels in cape town there is a thing, where all doves hates squirrels and all squirrels hate doves. I think it’s very important for us to create a little universe for these animals. The rules of their universe can be very similar to ours, but they are animals, so it will be subtly different, and that subtle difference will be the point of humour, because they will be so different, but so similar to us. Reading the article about the guy that made the viral dog videos we realised that what makes those videos funny is the fact that these dogs have human voices and human personalities, the fact that they sound human and not cartoony, makes it funny in a subtle way. Its almost like it contrasts against what we expect animals to sound like on videos. By making it serious it becomes funny, because humans dont always take animals seriously and we think that our lives are way more stressful than theirs you see. So if we make animals sound like they are stressing, then we laugh because we always tell ourselves animals have easy lives. Thats why we chose the urban animals, because these animals have it rough... look at what this squirrel is eating... some crappy human food in a plasitc bag, normal rural squirrels eat nice nuts... let's work with these out of the ordinary things and build on them. But more specifically lets make this cape townian above all. What are cape townians like? We want the squirrel to be a likeness of a cape townian so that if cape townians watch this they see themselves in a weird sort of way. 
 
This is quite alot, so hope you dont get confused in all the stuff i wrote! Oh before i forget, here is the link to the article about the guy that made the viral dog videos i told you about
https://www.fastcompany.com/3016431/how-to-make-a-talking-dog-video-that-will-be-seen-and-shared-by-millions

**Comment by ApheleleM** [comment]
When are you guys coming Back?

**Comment by CIEspost** [comment]
@ApheleleM sorry i never saw this comment, you need to tag me in the comment otherwise i dont get notified

**Comment by CIEspost** [comment]
@zacck please can you remember to make comments once you recieve emails and add the content of the email here if you see that no one else but you got the mail

**Comment by CIEspost** [comment]
@ApheleleM we have recieved a 1st draft of the voice over after we gave Josh a brief

**Comment by CIEspost** [comment]
I have saved the clip in the hapi hour file in my CIEproductions drive

**Comment by CIEspost** [comment]
@ApheleleM and @zacck please have a look at this and tell me what you think, i need to send comments trhough to Josh https://youtu.be/-eLAMdA5BhI

 and just want some more opinions, I personally feel it might be shorter and that he needs to speak as if the food is in his mouth

**Comment by CIEspost** [comment]
@ApheleleM @zacck Josh needs us to send him feedback asap he doesnt have much time 

I am busy editing the sound with these comments below in mind and then sending josh the sound: with these comments:

Hey Josh, thanks this is super awesome! You hit the nail on the head man! Really great! Just some things I'm thinking we can tweak perhaps, please let us know what you think about these points?
- So once he starts eating can you please try and sound like you have food in your mouth perhaps?
-  Really like the hadida comment, but think it needs to be at minute 00:57 when the hadida cries
- also put gaza vroom comment at moment that the first car vrooms
- put ty ty comment last

**Comment by ApheleleM** [comment]
Sorry for the late reply, 
Firstly..I like it but, I feel like the is no energy at all, and I would prefer the narration in English, But of course a bit of Afrikaans slag.
Secondly I dont think its short at all, but I do feel, like the action or emotions of the Squirrel does not go with the voice, or the character 
Thirdly, I wish, the Narration could happen as if the Squirrel is really speaking. Like those cat and Dogs video, narration must go with an action.
Lastly I think Josh, Need to be directed when, he does his Voice over,

**Comment by CIEspost** [comment]
@ApheleleM awesome thanks for the comments, i made some changes on the edit and the voice over, busy uplaoding the edit now. I will send Josh this with extra comments and ask him to add polishing touches to this. Will send through link to video once uplaoded

**Comment by CIEspost** [comment]
@ApheleleM please remember to tag me on comments with the @ sign and then my name so @carla or @zacck

**Comment by CIEspost** [comment]
@ApheleleM and @zacck here is the edited version please whatch and commment https://youtu.be/vo4HUSoKLAk (it will be done uplaoded in 5 min)

**Comment by CIEspost** [comment]
@zacck please add your comments here

**Comment by CIEspost** [comment]
i think we all agreed that the squirrel must talk faster so that the voice is more in sync with the animals movements and mouth

**Comment by CIEspost** [comment]
- speak as if food in his mouth
- sync speaking to body movement and gestures
- redo new script

**Comment by CIEspost** [comment]
sent Josh this 

Cool so firstly thanks so much for the awesome narration! We are so excited, its really getting close and looking awesome, now its just for the final push 👊✨ let us know when you have time to speak then we will call. But basically the idea is that with the videowe sent via mail you will hear that we changed the words around a bit so that he says somehting about the car vroom when we hear it vroom and he speaks about the hadida when the sound of the hadida is heard. And by the banana peel bit we edited it in a way so that it looks like that is what he is saying physically. We think this bit with the banana peel is perfect and doesnt require us to do it over, but then everything before and after that needs another take please. The changes that we are suggesting is that you please jumble the words around a bit (please refer to the edit we sent via email as a reference to how the words can be) as well as the way that you speak in a way that it syncs with the movement of the squirrel more, cause the squirrel moves so fast, perhaps speaking faster a bit more, maybe Ty Ty is high but paranoid haha. So technically we think maybe we make snippets for bits of movement rather than one long track? SO from here on we have the script now more or less, now its more up to us fitting the words and voice perfectly to the body movements and rythm of the squirrels actions. So when he is chewing as well it ,might work if you can chew physically too, so that it sounds like you have food in your mouth when you are talking, what do you think?

**Comment by zacck-zz** [comment]
👍  maybe we even break the video to segments and make a phrase for each segment then collage all it together. I think I agree with this current direction

**Comment by CIEspost** [comment]
@zacck we need to call Josh still

**Comment by zacck-zz** [comment]
👍  lets do this in the afternoon today.

**Comment by CIEspost** [comment]
@zacck please chat to josh on whatsap and ask him when is good for him to chat, perhaps organize that we chat at a time that he can be ready to record?

**Comment by CIEspost** [comment]
@josh said                         Hey guys I've got a deadline for the end of this week and I realise this is not as easy as I thought coz i need to  practice chewing with fast clear speech with remembering the punchlines mixing English and Afrikaans in a that accent so pls allow me to sort all the music out for my single this week... I will sort it out as promised as I only have some evenings free to record. Pls let me know if this is cool..."

**Comment by CIEspost** [comment]
@zacck i listened to your message, please tell Josh that he must let us know once he is ready then we can talk about the difficulties of his process and try and make sure we have comunicated the idea clearly so that we can make sure we are all on the same page, so maybe sunday or whenever is chill for him to talk about the way forward - i think between us the recording and video is still imperitve to be done asap, jsut as a fomrality so that we can make the budget for the series and carry on polishing the marketing strategy for hapihour so that we can have the documents raedy when needed to show to possibel investors

**Comment by CIEspost** [comment]
@zacck i am moving this issue to the future sprint then though cause iu assume Josh will only be working on this next week then

**Comment by zacck-zz** [comment]
@CIEspost that is correct

**Comment by CIEspost** [comment]
@zacck where is the other info on the feedback to Josh again, cant find it?

here's Tyrone's feedback 
cool so language don't understand it all.
am concerned it "might" not get an audience.
just giving my personal view as I would not watch it.
but u can always test it out.

**Comment by CIEspost** [comment]
@ApheleleM Josh just let us know that he is in town tomorow, so we can record the sound with him tomorw evening, if you are still able to come trhough after work, can you help us record with Josh tomorw night then please. You can sleep at our place if needed? Please let me know if you get this message, or when you are on lunch and i can call you.

**Comment by ApheleleM** [comment]
@CIEspost I can Help recording Josh, I will do sleep over, We still needs a lot to discuss.

**Comment by CIEspost** [comment]
@ApheleleM got that info a bit confused, Josh is here tonight, not tomorow, I'm going to call you at 12:30 that cool? Yes, lets discuss the rest that needs to be discussed on Saturday and on that issue please

**Comment by ApheleleM** [comment]
@CIEspost  I be waiting for the call

**Comment by CIEspost** [comment]
@zacck @ApheleleM see here for comments i made https://docs.google.com/document/d/1eJUvppRacVpXyXKbxR-DVfGYNAyj58JRSRwmrFn5Rp0/edit?usp=sharing

**Comment by CIEspost** [comment]
here is the new edit version we will show josh https://www.youtube.com/watch?time_continue=23&v=42TPgMQsU8E

**Comment by CIEspost** [comment]
we are using this song with the video now https://www.youtube.com/watch?v=8gyLR4NfMiI
carla will edit a new version with this song, editing the song to the squirrel
the dove will be chris brown, opening the track for the squirrel, who comes in at "yo breezy"

**Comment by CIEspost** [comment]
@ApheleleM it went well last nite, me and Josh got some good ideas and I am busy making a draft of the new script and will post it here once done
Josh will then record from the new script i give hime

**Comment by CIEspost** [comment]
Here's the lyrics to the song:

Ayo Breezy
Let me show you how to keep the dice rolling
When your doing that thing over there homie

Let's go!
'Cause I feel like I'm running
And I'm feeling like I gotta get away, get away, get away
Better know that I don't and I won't ever stop
'Cause you know I gotta win everyday day, day, go!
See they don't really wanna pop me
Just know that you never flop me
And I know that I can be a little cocky
You ain't never gonna stop me
Every time I come a nigga gotta set it,
Then I gotta go, and then I gotta get it
Then I gotta blow,

Let's go
See the way we on and we all up in the race and you know
We gotta go, don't try to keep up with the pace
We struggling and hustling and sending it and getting it
And always gotta take it to another place
Gotta taste it and I gotta grab it
And I gotta cut all through his traffic
Just to be at the top of the throne
Better know I gotta have it, have it

**Comment by CIEspost** [comment]
@ApheleleM and @zacck  
Josh is going to write a remix of this song and record his voice and make music
The song is just a reference here please let me know what you think of this?
https://youtu.be/gaeiJVebGwk

**Comment by CIEspost** [comment]
@zacck please watch this and give me feedback we need to send Josh an update ASAP! PLz i need advice here as I am quite stuck and need a second opinion

**Comment by zacck-zz** [comment]
@CIEspost  so I just watched this here are my observations.

1. Why is Chris Brown the dove any context to get from this?
2. I like that as the squirrel walks in Busta Rhymes gives a speech this serves as a connection for both.
3. The rap feels better than the conversations before maybe because of the energy matches.
4. The audio feels like it can match the squirrels actions better.

What's the squirrel's name?

**Comment by CIEspost** [comment]
We wil discus al this with aphelele Saturday 26 August and send Josh the new script by Saturday evening @zacck please can you send him a message tomrow and tell him we are sending him the update by end of the weekend

**Comment by zacck-zz** [comment]
👍  done!

**Comment by CIEspost** [comment]
@zacck thanks, I completetly forgot to discuss this with @ApheleleM , so am wondering if we can save some more time and wait to get her feedbaclk on Monday before we send Josh the new script

**Comment by zacck-zz** [comment]
@CIEspost  I will message him and let him know.

**Comment by CIEspost** [comment]
@ApheleleM please see the latest Idea me and Josh came up with and let me know what you think 
Josh is going to write a remix of this song and record his voice and make music
The song is just a reference here please let me know what you think of this?
https://youtu.be/gaeiJVebGwk

**Comment by CIEspost** [comment]
@ApheleleM commented: "its not funny enough, but I do like this it will make alot of sense once it looks likes the Squirrel, the one rapping"

**Comment by CIEspost** [comment]
I have made a new edit, @ApheleleM @zacck  please check here and give comments quick
Josh is going to write a remix of this song and record his voice and make music
The song is just a reference here please let me know what you think of this? https://www.youtube.com/watch?v=MQC57tXXu-o&feature=youtu.be

**Comment by CIEspost** [comment]
this is the final edit, here is the video link https://youtu.be/MQC57tXXu-o
and here is the lyrics to the parts that feature in the song

Look At Me Now (Explicit Version) (feat. Lil' Wayne & Busta Rhymes)
--
Chris Brown
Ayo Breezy
Let me show you how to keep the dice rolling
When your doing that thing over there homie
Let's go!
Cause I feel like I'm running
And I'm feeling like I gotta get away, get away, get away
Better know that I don't and I won't ever stop
Cause you know I gotta win everyday day, day, go!
See they don't really wanna pop me
Just know that you never flop me
And I know that I can be a little cocky
You ain't never gonna stop me
Every time I come a nigga gotta set it,
Then I gotta go, and then I gotta get it
Then I gotta blow,
 
And then I gotta shudder any little thing that nigga think he be doing
Cause it doesn't matter, 'cause I'm gonna ba ba ba ba
bang beats on my chest,
And I bang in the east, and I'm banging in the west
And I come to give you more and I will never give you less
You will hear it in the street or you can read it in the press
Do you really wanna know what's next? Let's go
See the way we on and we all up in the race and you know
We gotta go, don't try to keep up with the pace
We struggling and hustling and sending it and getting it
And always gotta take it to another place
Gotta taste it and I gotta grab it
And I gotta cut all through his traffic
Just to be at the top of the throne
Better know I gotta have it, have it
 
Look at me now, look at me now
Oh, I'm getting paper
Look at me now
Oh, look at me now
Yeah, fresher than a motherfucker
 
Man fuck these bitch ass niggas, how y'all doin'?
I'm Lil Tunechi, I'm a nuisance,
I go stupid, I go dumb like the three stooges
I don't eat sushi, I'm the shit, no I'm pollution, no substitution
Got a bitch that play in movies in my Jacuzzi, pussy juicy
I never gave a fuck about a hater, got money on my radar
Dress like a skater, got a big house, came with an elevator
You niggas ain't eatin', fuck it, tell a waiter
Marley said, "Shoot 'em", and I said, "Okay"
If you wanted bullshit then I'm like ole
I don't care what you say, so don't even speak
 
Your girlfriend a freak like Cirque Du Soleil
That's word to my flag, and my flag red
I'm out of my head, bitch I'm outta my mind, from the bottom I climb
You ain't hotter than mine, nope, not on my time and I'm not even trying
What's poppin' Slime? Nothin' five, and if they trippin' fuck 'em five
I ain't got no time to shuck and jive, these niggas as sweet as pumpkin pie
Ciroc and Sprite on a private flight, bitch I been tight
Since guiding light, and my pockets white, and my diamonds white
And my mommas nice and my daddy's dead
You fagots scared 'cause I'm too wild, been here for a while
I was like fuck trial I puts it down
I'm so Young Money, if you got eyes look at me now, bitch
 
Look at me now, look at me now
Oh, I'm getting paper
Look at me now
Oh, look at me now
Yeah, I'm fresher than a motherfucker

**Comment by CIEspost** [comment]
Instructions to Josh:

Here's the final video, so you can hear we mashed it up a bit. 

But first the instructions:
So if you can please use these lyrics and change the words and the accent like a cape town remake of the song as we discussed and agreed last
As you can see I have included some of Chris Brown's words in the end, we decided to rather make it sound liek the squirrel says that too instead of using the actual mp3 of the real track. SO... all the sound will be remade, most of it will be you rapping, but then 
if you can also please add music and sounds like we discussed last then that will be great! 

Then lastly, can you please send me the rapping and musical bits in separate layers then I will edit them to fit with the video.

Here's the lyrics:


"Ayo Breezy		
Let me show you how to keep the dice rolling		
When your doing that thing over there homie		
Let's go!		
Cause I feel like I'm running		
And I'm feeling like I gotta get away, get away, get away		
Better know that I don't and I won't ever stop		
Cause you know I gotta win everyday day, day, go!		
See they don't really wanna pop me		
Just know that you never flop me		
And I know that I can be a little cocky		
You ain't never gonna stop me		
Every time I come a nigga gotta set it,		
Then I gotta go, and then I gotta get it		
Then I gotta blow,		
		
And then I gotta shudder any little thing that nigga think he be doing		
Cause it doesn't matter, 'cause I'm gonna ba ba ba ba		
bang beats on my chest,		
And I bang in the east, and I'm banging in the west		
And I come to give you more and I will never give you less		
You will hear it in the street or you can read it in the press		
Do you really wanna know what's next? Let's go		
See the way we on and we all up in the race and you know		
We gotta go, don't try to keep up with the pace		
We struggling and hustling and sending it and getting it		
And always gotta take it to another place		
Gotta taste it and I gotta grab it		
And I gotta cut all through his traffic		
Just to be at the top of the throne		
Better know I gotta have it, have it		
		
Look at me now, look at me now		
Oh, I'm getting paper		
Look at me now		
Oh, look at me now		
Yeah, fresher than a motherfucker		
		
Man fuck these bitch ass niggas, how y'all doin'?		
I'm Lil Tunechi, I'm a nuisance,		
I go stupid, I go dumb like the three stooges		
I don't eat sushi, I'm the shit, no I'm pollution, no substitution		
Got a bitch that play in movies in my Jacuzzi, pussy juicy		
I never gave a fuck about a hater, got money on my radar		
Dress like a skater, got a big house, came with an elevator		
You niggas ain't eatin', fuck it, tell a waiter		
Marley said, "Shoot 'em", and I said, "Okay"		
If you wanted bullshit then I'm like ole		
I don't care what you say, so don't even speak		
		
Your girlfriend a freak like Cirque Du Soleil		
That's word to my flag, and my flag red		
I'm out of my head, bitch I'm outta my mind, from the bottom I climb		
You ain't hotter than mine, nope, not on my time and I'm not even trying		
What's poppin' Slime? Nothin' five, and if they trippin' fuck 'em five		
I ain't got no time to shuck and jive, these niggas as sweet as pumpkin pie		
Ciroc and Sprite on a private flight, bitch I been tight		
Since guiding light, and my pockets white, and my diamonds white		
And my mommas nice and my daddy's dead		
You fagots scared 'cause I'm too wild, been here for a while		
I was like fuck trial I puts it down		
I'm so Young Money, if you got eyes look at me now, bitch		
		
Look at me now, look at me now		
Oh, I'm getting paper		
Look at me now		
Oh, look at me now		
Yeah, I'm fresher than a motherfucker

**Comment by CIEspost** [comment]
Josh sent us a message back saying he is quite busy right now
